src/components/ToolbarButton/ToolbarButton.module.scss

```diff
@@ -0,0 +1,72 @@
+.toolbarButton {
+  box-sizing: content-box;
```

Comment:
Changing `box-sizing` to something other than `border-box` is generally frowned upon, because `content-box` changes the element's size in a way we often don't expect (padding makes the element larger). Is it necessary to use `content-box` here, and if so, can we document why?

Reply:
content-box was previously used in VT360 and that is maybe because of the icon font that was used. But I agree that we can change it to border-box